content/pages/healthcare/health-conditions/mental-health.md

```diff
@@ -122,3 +133,16 @@ You may still be able to get care:
 **Need more information and support?** <br />
 [Find more resources for help in a crisis](http://www.mentalhealth.va.gov/gethelp.asp).<br /> 
 [Access additional resources for ongoing support](http://www.mentalhealth.va.gov/Resources.asp). 
+
+<script type="text/javascript">
+
+  function toggleClass(elementId, className) {
+    document.getElementById(elementId).classList.toggle(className);
+  }
+
+  // Toggle the expandable crisis info
+  document.getElementById('crisis-expander-link')
+    .addEventListener('click', function () {
+      toggleClass('apply-expander-content', 'expander-content-closed');
```

Comment:
Because we're not re-using `toggleClass()`, if you want to save space, you can replace this line with
```js
document.getElementById('apply-expander-content').classList.toggle('expander-content-closed');
```
and delete the `toggleClass()` function above.